src/pages/RegisterToken/hooks/useRegisterTokenForm.ts

```diff
@@ -84,7 +82,11 @@ export const useRegisterTokenForm = () => {
       [RegisterTokenFormFieldsEnum.type]: string().required('Type is required')
     }),
     onSubmit: async (values) => {
-      const token = tokens.find((t) => t.identifier === values.token.value);
+      const token = tokens.find((t) =>
```

Comment:
{identifier, ticker}

Reply:
'Identifier' may not be defined. Does not work